### PR TITLE
[Console] Add constants for main exit codes

### LINKED ISSUE #35431: [Console] Define constants for main exit codes.
**Description**  
Console commands must return exit code. I propose we define constants for most used exit statuses like it is done in C programming language.
https://www.gnu.org/software/libc/manual/html_node/Exit-Status.html
That would make code a bit more clear.

**Example**  
Before:
`return 0;`
After:
`return self::EXIT_SUCCESS;`

### PR DESCRIPTION
**Pull request by Chi-teck**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | yes
| Deprecations? |no
| Tickets       | Fix #35431
| License       | MIT

### CODE REVIEW

**Comment by Chi-teck** [comment]
Can we use these constants in console's own code?

**Review by javiereguiluz — APPROVED** [review]
I'd prefer `EXIT_ERROR` instead of `EXIT_FAILURE` ... but "failure" seems to be the standard word for this (e.g. https://www.gnu.org/software/libc/manual/html_node/Exit-Status.html)

Thanks @Chi-teck!

**Comment by fabpot** [comment]
What about just `Command::SUCCESS` and `Command::FAILURE`?
I would not use these constants in Symfony's code.

File: src/Symfony/Component/Console/Command/Command.php [hunk 1]
```diff
@@ -29,6 +29,9 @@
  */
 class Command
 {
+    const SUCCESS = 0;
```
Comment (OskarStark) [inline src/Symfony/Component/Console/Command/Command.php]: public const 👍🏻

**Review by OskarStark — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @Chi-teck.

**Pull request merged**